### PR TITLE
Add SignUp and Home activity

### PR DESCRIPTION
**Pull request by camilo05**
Sep-10-2021 00-37-38

Sep-10-2021 14-50-51

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeView.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package ar.com.wolox.android.example.ui.home
+
+interface HomeView
```
Comment (rodrigo103) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomeView.kt]: Usually we declare the view interface inside the fragment file. something like this:
```
class HomeFragment : WolmoFragment<FragmentHomeBinding, HomePresenter>(), HomeView {
    ...
}

interface HomeView {
    ...
}
```

File: app/src/main/res/layout/fragment_signup.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+
```
Comment (rodrigo103) [inline app/src/main/res/layout/fragment_signup.xml]: Delete this empty line, one is enough.

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+
```
Comment (rodrigo103) [inline app/src/main/res/layout/fragment_home.xml]: Delete this empty line, one is enough.

**Comment by rodrigo103** [comment]
Looks great!! I left some small comments

**Review by rodrigo103 — COMMENTED** [review]
Looks great!! I left some small comments

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+class HomeActivity @Inject constructor() : WolmoActivity<ActivityBaseBinding>() {
+    override fun layout() = R.layout.activity_base
+
+    override fun init() {
+        replaceFragment(binding.activityBaseContent.id, HomeFragment.newInstance())
+    }
+
+    companion object {
+        fun start(context: Context) = context.jumpTo(HomeActivity::class.java)
```
Comment (LisandroDiMeo) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt]: Keep in mind what `jumpTo` does, read its implementation 👍

**Review by LisandroDiMeo — APPROVED** [review]
Looks good! Just one comment to keep in mind theimplementation of a method

**Review by ffasitella — CHANGES_REQUESTED** [review]
I have just one comment: What happens when the user logs in, kills the app and then opens it again? The app should contain the logic to detect if the user is logged (reading UserSession) and redirect to the Home or the Login.

**Review by ffasitella — APPROVED** [review]
Amazing! 🚀

**Review by rodrigo103 — APPROVED** [review]
Well done!!

**Pull request merged**